jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-6a.yaml

```diff
@@ -159,7 +159,7 @@
           env | grep -e networking -e libvirt -e cloud > mkcloud.config
 
           echo "want_magnum=1" >> mkcloud.config
-          echo "ostestroptions=\"--regex '^magnum.tests.functional.api'\"" >> mkcloud.config
+          echo "ostestroptions=\"--regex '^magnum.tests.functional.k8s'\"" >> mkcloud.config
```

Comment:
API functional tests have hardcoded swarm as COE backend, which we don't support. There is no sense to run this tests any more.